src/commands/project/serve-web.ts

```diff
@@ -22,6 +22,7 @@ export default class ProjectServeWeb extends NimBaseCommand {
   static flags = {
     namespace: flags.string({ description: 'The namespace to proxy (current namespace if omitted)' }),
     apihost: flags.string({ description: 'API host of the namespace' }),
+    port: flags.integer({ description: 'The port of the web server' }),
```

Comment:
```suggestion
    port: flags.integer({ description: 'The port of the web server (default is 8080)' , default: 8080}),
```

Reply:
You can define the default on the flag itself rather than `const port = flags.port || 8080`